bin/edit_user_role.ts

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
Is this a script you have used on the production db?

Reply:
No. It started off as a script to edit user details but became too complicated, so I just changed it to manage roles. It's similar to #288 but takes arguments.

Reply:
Can also just delete though 🤷🏼‍♂️

Reply:
Up to you. Keep it if you think it’ll be useful in future.

Reply:
Lets keep it for now then